certs/handGrenades.md

```diff
@@ -0,0 +1,34 @@
+# Grenade Certification
+
+## Instructors
+
+- Brett
```

Comment:
You can put yourself here too

Reply:
Hahha I was messing around and accidentally hurt myself and didn't bother to heal. But I can change that and get rid of the Identifier files as well. Sorry didn't realize those snuck in there. Wtf are those?